terraform/gcp/projects/pilot-hubs.tfvars

```
@@ -35,6 +35,15 @@ notebook_nodes = {
     resource_labels : {
       "community" : "climatematch"
     }
+  },
+  "neurohackademy" : {
+    # We expect around 120 users
+    min : 1,
+    max : 100,
+    machine_type : "n1-highmem-n16",
```

Always go for the cheaper per RAM and more cost performant per CPU `n2-` machines.

```suggestion
    machine_type : "n2-highmem-16",
```

---

Do we have that documented somewhere? I will just doing a manual reversion of a PR (because it couldn't automatically be applied from the UI)

---

Not yet I think, I've done a lot of work towards standardizing towards `n2-highmem` and `r5` machines (highmem) on AWS though - updating various cluster templates and such so new clusters will end up with the modern machine types at least.

---

Related to https://github.com/2i2c-org/infrastructure/issues/2121#issuecomment-1497387131

---

I guess it doesn't matter too much right now. But also, this was supposed to be the easier option as it was "revert two PRs" rather than setting up something new on AWS 😅